roles/seqreports/tasks/main.yml

```diff
@@ -1,30 +1,5 @@
 ---
 
-- name: create conda environment for seqreports
-  shell: "{{ anaconda_path }}/bin/conda create -n {{ seqreports_venv_name }} -c anaconda python=3.8 -y"
-  args:
-    creates: "{{ anaconda_path }}/envs/{{ seqreports_venv_name }}/bin/python3"
-
-- name: Set host fact boolean for miarka deployment
-  set_fact:
-    miarka_deployment: "{{ miarka_deployment | default(true) }}"
-
-- name: ensure seqreports data path exists
-  file:
-    path: "{{ seqreports_data_path }}"
-    state: directory
-
-- name: ensure nextflow directories exist
-  file:
-    path: "{{ item }}"
-    state: directory
-  loop:
-    - "{{ seqreports_nxf_temp }}"
-    - "{{ seqreports_nxf_work }}"
-    - "{{ seqreports_nxf_logs }}"
-
-- import_tasks: pre_reqs.yml
-
-- name: Update and deploy checkQC config
-  import_tasks: deploy_config.yml
-
+- block:
+  - include: 1_install.yml
```

Comment:
"include" is still working, but is deprecated. Please use "include_tasks" instead.